guide/creating-your-bot/common-questions.md

```diff
@@ -109,6 +109,12 @@ user.send('<content>');
 
 <p class="tip">If you want to learn more about this syntax or want to learn about reaction collectors as well, check out [this dedicated guide page for collectors](/creating-your-bot/collectors)!</p>
 
+```js
+<message>.channel.send('My message to react to.').then( msg => {
+	msg.react('👍');
```

Comment:
* There should be a short question above the code snippet, similar to the rest of the snippets (like "How do I react a message my bot sends?")
* No space before `msg =>`
* This is mostly just a personal preference, but I'd rename `msg` to `sent` or `sentMessage`, because ithelps avoid name conflicts with those who use `msg` as their main variable, since we use `<message>` as a placeholder, and it also brings clarity when reading the code. Again, just a personal preference, but worth mentioning.